amgut/handlers/auth_handlers.py

```diff
@@ -23,6 +23,8 @@ def get(self):
     @authenticated
     def post(self):
         skid = self.current_user
+        # log user out for register process
+        self.clear_cookie("skid")
         tl = text_locale['handlers']
         info = {}
         for info_column in ("email", "participantname", "address", "city",
```

Comment:
we should not allow `participantname` to be `None`. This loop is getting default values, but aren't these fields required?